### PR TITLE
add gitui discord server

### PR DESCRIPTION
**Pull request by extrawurst**

### CODE REVIEW

File: communities.json [hunk 1]
```diff
@@ -771,6 +771,12 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "gitui.png",
```
Comment (jamiebuilds) [inline communities.json:776]: After merging, your project will be featured on our [Discord ❤ Open Source](https://discord.com/open-source) page. It automatically applies a filter to your logo that makes it solid white except for the transparent portions.

The image you've included will look incorrect when this filter is applied. Could you swap the image out for a version with a transparent background that will look correct when solid white on a dark background?

**Comment by jamiebuilds** [comment]
Turned on some features for you:

- [Vanity Invite URL](https://support.discord.com/hc/en-us/articles/115001542132-Server-Vanity-URLs)
- [Server Banner & Invite Splash](https://support.discord.com/hc/en-us/articles/360028716472-Server-Banner-Background-Invite-Splash-Image)

Do you want to update your invite code to something like `discord.gg/<yourprojectname>`?

File: communities.json (hunk 1 above)
Comment (extrawurst) [reply communities.json:776]: ah ok I checked out the worst possible example: `tokio.png`
anyhow it should be white on transparent background now

**Comment by extrawurst** [comment]
@jamiebuilds should be good to go now

**Review by jamiebuilds — APPROVED** [review]

**Comment by extrawurst** [comment]
@jamiebuilds what do I need to do to merge this?

**Pull request closed without merging**